library/src/main/java/com/novoda/downloadmanager/DownloadError.java

```diff
@@ -33,35 +45,37 @@ public String message() {
         return message;
     }
 
+    @Nullable
+    public Integer violatedRuleCode() {
+        return violatedRuleCode;
+    }
+
     @Override
     public boolean equals(Object o) {
-        if (this == o) {
-            return true;
-        }
-        if (o == null || getClass() != o.getClass()) {
-            return false;
-        }
+        if (this == o) return true;
```

Comment:
probably it's safer to keep the original formatting style

Reply:
Good point. I've updated the bracing since.